src/modules/wallet/web3-react/connection/walletConnect.tsx

```diff
@@ -42,7 +42,7 @@ const [web3WalletConnect, web3WalletConnectHooks] = initializeConnector<WalletCo
       options: {
         rpc: RPC_URLS,
         qrcode: true,
-        bridge: 'https://safe-walletconnect.safe.global',
+        bridge: process.env.REACT_APP_WALLET_CONNECT_V1_BRIDGE || 'https://safe-walletconnect.safe.global',
```

Comment:
If we want this to be settable in any way other than the .env files (meaning, Vercel), add it here:

https://github.com/cowprotocol/cowswap/blob/76bf689bad5f6b914e0a2a5a5180fab5babfe618/.github/workflows/vercel.yml#L35-L43

Keep in mind that doing that overwrites .env values

Reply:
it worked when i changed the env in vercel. Anyways, adding this here

Looking FW we get rid of these logics, its too hard to add an env 😢

Reply:
done 7934e5ebd405707ab50eb911d70ae7fc0ee4aa01

Reply:
> it worked when i changed the env in vercel.
Yes, that should work, for PR deployments.

Every other env (dev, staging, barn and prod) use env vars from Github, NOT Vercel, thus it needs the GH action change